runtime/domain.c

```diff
@@ -373,15 +392,26 @@ asize_t caml_norm_minor_heap_size (intnat wsize)
   and if a request comes to change the minor heap size.
 
   The boundaries of this committed memory area are
-    [domain_state->young_start]
-  and
-    [domain_state->young_end].
+     [domain_state->young_start]
+   and
+     [domain_state->young_end].
+
+  Those [young_{start,end}] variables are never accessed by another
+  domain, so they need no synchronization.
 */
-int caml_reallocate_minor_heap(asize_t wsize)
-{
+
+void caml_check_minor_heap() {
```

Comment:
This function is not used outside this file. Inline it.

```suggestion
Caml_inline void check_minor_heap () {
```